sample-compose/app/src/test/java/co/nimblehq/sample/compose/ui/screens/home/HomeScreenTest.kt

```diff
@@ -56,6 +56,28 @@ class HomeScreenTest : BaseScreenTest() {
         every { mockIsFirstTimeLaunchPreferencesUseCase() } returns flowOf(false)
     }
 
+    @Test
+    fun `When entering the Home screen for the first time, it shows a toast confirming that`() {
+        every { mockIsFirstTimeLaunchPreferencesUseCase() } returns flowOf(true)
+
+        initComposable {
+            composeRule.waitForIdle()
+            advanceUntilIdle()
+
+            ShadowToast.showedToast(activity.getString(R.string.message_first_time_launch)) shouldBe true
```

Comment:
My last concern @AVI5HEK, Let's verify `UpdateFirstTimeLaunchPreferencesUseCase` as well 🚀 
```suggestion
            ShadowToast.showedToast(activity.getString(R.string.message_first_time_launch)) shouldBe true

            coVerify(exactly = 1) {
                mockUpdateFirstTimeLaunchPreferencesUseCase(false)
            }
```

Reply:
@Wadeewee IMO, that's not a concern of the UI. We have that covered in the ViewModel test.

Reply:
I think it is good to have it but if you think it's unnecessary, So it's optional for you to update then 🚀